services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/grpc/GrpcCommonSpec.groovy

```diff
@@ -1,12 +1,12 @@
 package org.openkilda.functionaltests.spec.grpc
 
-import groovy.util.logging.Slf4j
+import spock.lang.Unroll
 
-@Slf4j
 class GrpcCommonSpec extends GrpcBaseSpecification {
-    def "Able to get switch status"() {
+    @Unroll
+    def "Able to get switch status on the #switches.address switch"() {
```

Comment:
switch `address` is not a common way to identify switch. I'd rather it was switchId or switch name

Reply:
`changed to switchId`